test/suite/test_hs01.py

```diff
@@ -150,12 +168,22 @@ def test_hs(self):
         bigvalue4 = b"ddddd" * 100
         self.conn.set_timestamp('stable_timestamp=' + self.timestamp_str(1))
         self.large_updates(self.session, uri, bigvalue4, ds, nrows, timestamp=True)
-        # Check to see data can be see only till the stable_timestamp
-        self.durable_check(bigvalue3, uri, ds, nrows)
+
+        self.session.checkpoint()
+        # Check if the 9999 modifications were moved to history store from data store.
```

Comment:
Thanks for the change the test to use `nrows`. Can you adjust the comments also according?